server/game/gameSystems/MoveCardSystem.ts

```diff
@@ -105,18 +105,21 @@ export class MoveCardSystem<TContext extends AbilityContext = AbilityContext> ex
     public override canAffect(card: Card, context: TContext, additionalProperties = {}, mustChangeGameState = GameStateChangeRequired.None): boolean {
         const { destination } = this.generatePropertiesFromContext(context, additionalProperties) as IMoveCardProperties;
 
+        // Token cards can be attempted to be moved anywhere
+        if (!card.isToken()) {
```

Comment:
We can still have a destination check for tokens. I think there's no reason an effect should move a token to the base zone, that would be an error we can check with Contract.assertTrue. I think it's legal to attempt to move them basically anywhere else though.

Reply:
Added the `Contact.assertTrue`. Is this what you were thinking?